src/cf.data.pre

```diff
@@ -1422,6 +1422,10 @@ endif
 	  # adaptation_meta because it starts matching immediately after
 	  # the service has been selected for adaptation.
 
+	acl aclname adaptation_rep_header header-name [-i] any\.regex\.here
+	  # regex match against the adaptation reply headers.  e.g. ICAP
+	  # reply headers. [fast]
```

Comment:
Please document that the ACL does not match if the master transaction has seen no responses from adaptation services. Compared to HTTP headers, this is somewhat unusual, because there are always some HTTP headers (or at least that is how most folks think of HTTP-related ACLs). This new ACL will not match during http_access, for example (unless SslBump is in use), even if REQMOD adaptations are enabled for all HTTP requests.

Reply:
Small modification to the request. Please do not use the term "master". There is only one type of transaction that can be passed to adaptation by Squid - an "HTTP transaction".

Reply:
This ACL does not match any specific HTTP transaction. It matches a master transaction that may include two or more HTTP transactions.